src/modules/Builder/features/Activities/Activities.tsx

```diff
@@ -105,7 +97,7 @@ export const Activities = () => {
         ? performanceTaskName
         : t('newActivity');
 
-    const name = getAddedActivityName(newActivityName, activityNames, setActivityNames);
+    const name = getUniqueName(newActivityName, pluck(activities, 'name'));
```

Comment:
maybe use separate variable for pluck(activities, 'name')?

Reply:
done